apps/api/src/movies/entities/movie.entity.ts

```diff
@@ -1,2 +1,21 @@
-/* eslint-disable @typescript-eslint/no-extraneous-class */
-export class Movie {}
+import { createZodDto } from 'nestjs-zod'
+import { z } from 'nestjs-zod/z'
+
+const MovieSchema = z.object({
+  id: z.number(),
+  genre: z.string(),
+  director: z.string(),
+  subtitles: z.string().array().optional(),
+  releaseDate: z.date(),
+  movieTitles: z.array(z.object({
+    id: z.number(),
+    title: z.string(),
+    language: z.string(),
+    createdAt: z.date(),
+    updatedAt: z.date()
+  })),
+  createdAt: z.date(),
+  updatedAt: z.date()
+})
```

Comment:
Now that we have the Movie entity defined, thanks to you, we can use the Serializer for both API and DB implementations 🙌